src/AppRoutes.tsx

```diff
@@ -27,6 +28,7 @@ const AppRoutes = () => {
     <Suspense fallback={<DelayedFallback />}>
       <Switch>
         <Route exact path="/" component={Explore} />
+        <Route exact path="/handlenotfound" component={HandleNotFound} />
```

Comment:
skulle vi kanskje tatt med en cypress-test, bare for å passe på at vi ikke roter bort siden ?

Reply:
Fikset